File: client/client/DTO/Offer/ShortOfferResponseDto.cs
namespace client.DTO.Offer;

public class ShortOfferResponseDto
{
    public string Id { get; set; }
    public string Title { get; set; }
    public int OriginalPrice { get; set; }
    public int OfferPrice { get; set; }
    public int NumberOfAvailableItems { get; set; }
    public DateTime PickupTimeStart { get; set; }
    public DateTime PickupTimeEnd { get; set; }
    public string ImagePath { get; set; }
    public string Status { get; set; }
}

File: server/src/main/java/com/example/server/dto/offer/ShortOfferResponseDto.java
package com.example.server.dto.offer;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ShortOfferResponseDto
{

  private String id;
  private String title;
  private int originalPrice;
  private int offerPrice;
  private int numberOfAvailableItems;
  private LocalDateTime pickupTimeStart;
  private LocalDateTime pickupTimeEnd;
  private String imagePath;
  private String status;

}

